### ISSUE TITLE
Feature Request: python module

### ISSUE THREAD

**Issue by hex2a** [yadayada/acd_cli#405]
please also provide a python module for own python applications. (without fuse, etc; upload/downloads/folders controlled in the application)

**Comment by yadayada** [comment]
I'm not sure I understand what you're getting at. Could you please clarify? Do you want a separate package containing just the API?

**Comment by hex2a** [comment]
yes, a package with just the API. (as simple as possible, eg: without caching in sqlite)
